README.md

```diff
@@ -56,6 +56,20 @@ Import the `loadScript` function for asynchronously loading the Paypal JS SDK.
 -   accepts an object for passing query parameters and attributes to the JS SDK.
 -   returns a Promise that resolves with `window.paypal` after the JS SDK is finished loading.
 
+#### Async/Await
+```js
```

Comment:
Awesome first PR @wsbrunson! 💯 🎸

Can you run `npm run format` locally? I think there is some minor formatting tweaks that prettier will run (I thought this would get auto-formatted by the precommit hook but did not). Please push that up and then this should be good to merge!

Reply:
I hadn't run `npm install` yet so it committed without running any of the pre-commit hooks. Should be good now!